pkg/migrate/migrate.go

```diff
@@ -301,7 +321,7 @@ func copyOnePVC(ctx context.Context, w *log.Logger, clientset k8sclient.Interfac
 			return fmt.Errorf("logs for the migration pod %s in %s ended, but the status was %s and not succeeded", createdPod.Name, ns, gotPod.Status.Phase)
 		}
 
-		time.Sleep(time.Second * 5)
+		time.Sleep(waitTime * 5)
```

Comment:
why not have `waitTime` be set to `time.Second * 5` in the calling function?

Reply:
because it's not the same everywhere in this function
it might be good to just have everything use the same sleep though